src/sidebar/util/test/term-before-position-test.js

```diff
@@ -0,0 +1,62 @@
+import { termBeforePosition } from '../term-before-position';
+
+describe('term-before-position', () => {
+  // To make these tests more predictable, we place the `$` sign in the position
+  // to be checked. That way it's easier to see what is the "word" preceding it.
+  // The test will then get the `$` sign index and remove it from the text
+  // before passing it to `termBeforePosition`.
+  [
+    // Position in the middle of words
+    {
+      text: 'Hell$o world',
+      expectedTerm: 'Hell',
+    },
```

Comment:
Can you add test cases for when the position is at a) the start and b) the end of the string.